app/enterprise/changelog.md

```diff
@@ -4,6 +4,223 @@ no_search: true
+  Before disabled targets were not included in the output when users attempted to list all targets. Then
+  when users attempted to add the targets again, they received an error message telling them the targets already existed.
+  [#7094](https://github.com/kong/kong/pull/7094)
+- Upserting existing targets no longer fails.  Before, because of updates made to target configurations since Kong v2.2.0,
+  upserting older configurations would fail. This fix allows older configurations to be imported.
+  [#7052](https://github.com/kong/kong/pull/7052)
+- The last balancer attempt is now correctly logged. Before balancer tries were saved when retrying, which meant the last
+  retry state was not saved since there were no more retries. This update saves the failure state so it can be correctly logged.
+  [#6972](https://github.com/kong/kong/pull/6972)
+- Kong now ensures that the correct upstream event is removed from the queue when updating the balancer state.
+  [#7103](https://github.com/kong/kong/pull/7103)
+
+#### CLI
+- The `prefix` argument in the `kong stop` command now takes precedence over environment variables,
+  as it does in the `kong   start` command. [#7080](https://github.com/kong/kong/pull/7080)
+
+#### Configuration
+- Declarative configurations now correctly parse custom plugin entities schemas with attributes called "plugins". Before
+  when using declarative configurations, users with custom plugins that included a "plugins" field would encounter startup
+  exceptions. With this fix, the declarative configuration can now distinguish between plugins schema and custom plugins fields.
+  [#7412](https://github.com/kong/kong/pull/7412)
+- The stream access log configuration options are now properly separated from the HTTP access log. Before when users
+  used Kong with TCP, they couldn’t use a custom log format. With this fix, `proxy_stream_access_log` and `proxy_stream_error_log`
+  have been added to differentiate stream access log from the HTTP subsystem. See
+  [`proxy_stream_access_log`](/gateway-oss/2.5.x/configuration/#proxy_stream_access_log)
+  and [`proxy_stream_error`](/gateway-oss/2.5.x/configuration/#proxy_stream_error) in the Configuration
+  Property Reference for more information. [#7046](https://github.com/kong/kong/pull/7046)
+
+#### Migrations
+- Kong no longer assumes that `/?/init.lua` is in the Lua path when doing migrations. Before, when users created
+  a custom plugin in a non-standard location and set `lua_package_path = /usr/local/custom/?.lua`, migrations failed.
+  Migrations failed because the Kong core file is `init.lua` and it is required as part of `kong.plugins.<name>.migrations`.
+  With this fix, migrations no longer expect `init.lua` to be a part of the path. [#6993](https://github.com/kong/kong/pull/6993)
+- Kong no longer emits errors when doing `ALTER COLUMN` operations in Apache Cassandra 4.0.
```

Comment:
Let's add a note here that even with this, Cassandra 4.0 is not yet supported

Reply:
will do